guides/basic-use/cli-commands.md

```diff
@@ -102,23 +102,33 @@ ember generate <type-of-file> <name-of-your-choice>
 
 ### What it does
 
-`ember generate` creates new files within your app. For example, you can use it to create components, routes, services, models, and more. For a full list, type `ember generate --help`.
+`ember generate` creates new files within your app. For example, you can use it to create components, routes, services, models, and more.
+
+**For a full list, type `ember generate --help`.**
```

Comment:
made it as a bold separate line to emphasis. I find it hard to skim in the before version. Your thought @jenweber ?